common/src/main/scala/explore/components/Tile.scala

```diff
@@ -21,6 +22,7 @@ import react.semanticui.collections.menu._
 import react.semanticui.elements.button.Button
 
 final case class Tile(
+  id:                Tile.TileId,
```

Comment:
What is the `id` used for?

Reply:
It is the same id used in the grid, it makes some bits of the code simpler

One issue I gave is there is a certain duplication of states, this PR removes one by making the state of the tile dependent on the height